From 884133421724f55163e30ff0928430401fa6daca Mon Sep 17 00:00:00 2001
From: "Xu,Baochu" <quic_bxu@quicinc.com>
Date: Wed, 19 Jan 2022 10:26:30 +0800
Subject: [PATCH] fix mixer_get_array failed issue

---
 src/mixer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/git/src/mixer.c b/git/src/mixer.c
index f1b5a30..2ee3b36 100644
--- a/git/src/mixer.c
+++ b/git/src/mixer.c
@@ -423,7 +423,7 @@ struct mixer_ctl *mixer_get_ctl_by_name_and_index(struct mixer *mixer,
  */
 void mixer_ctl_update(struct mixer_ctl *ctl)
 {
-    ioctl(ctl->mixer->fd, SNDRV_CTL_IOCTL_ELEM_INFO, ctl->info);
+    ioctl(ctl->mixer->fd, SNDRV_CTL_IOCTL_ELEM_INFO, &ctl->info);
 }

 /** Checks the control for TLV Read/Write access.
--
2.17.1

